### ISSUE TITLE
add jest 

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/loopback-tutor-intern-10#10]
for testing custom methods api endpoints - not for all generated endpoints.
not finished task - can be discussed in comments

**Comment by louisevdb84** [comment]
Can Jest only be used with React Applications?

**Comment by atherdon** [comment]
not sure, but i think it works with nodejs stuff. did you check links from
this article:
https://medium.com/groceristar/things-that-should-read-and-use-javascript-intern-at-groceristar-april-18-collection-bd6541e9ae28

i'm newbie with jest too, but decided to switch from mocha because we're
moving from expressJs frontend to react


On 07-May-2018 at 10:10 AM, Louise wrote:


Can Jest only be used with React Applications?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ChickenKyiv/loopback-tutor-intern-10/issues/10#issuecomment-387020811>,
or mute the thread

**Comment by louisevdb84** [comment]
Yes, I read the articles and it only mentions redux to test actions and React. 

I like this about Jest:
"It automatically runs tests in parallel, and its watch mode is able to run only tests relevant to the changed file, which is invaluable when you have a large suite of tests. It comes with JSDom configured, meaning you can write browser tests but run them through Node, can deal with asynchronous tests and has advanced features such as mocking, spies and stubs built in." quoted from this article https://www.sitepoint.com/test-react-components-jest/

**Comment by atherdon** [comment]
Ok, i think jest will work for loopback api
